test-three/side-menu-app/src/components/SideMenu.js

```diff
@@ -0,0 +1,101 @@
+import React from 'react';
+import { makeStyles } from '@material-ui/core/styles';
+import ListSubheader from '@material-ui/core/ListSubheader';
+import List from '@material-ui/core/List';
+import Logo from '../images/Logo.png';
+import SearchBar from "material-ui-search-bar";
+import Paper from '@material-ui/core/Paper';
+import InputBase from '@material-ui/core/InputBase';
+// icons (material ui)
+import IconButton from '@material-ui/core/IconButton';
+import SearchIcon from '@material-ui/icons/Search';
+import DirectionsIcon from '@material-ui/icons/Directions';
+import HelpOutlineOutlinedIcon from '@material-ui/icons/HelpOutlineOutlined';
+import PersonOutlineOutlinedIcon from '@material-ui/icons/PersonOutlineOutlined';
+// components
+import CollapsableMenuItem from './CollapsableMenuItem';
+import SimpleMenuItem from './SimpleMenuItem';
+import '../styles/style.css';
+
+
+
+const useStyles = makeStyles((theme) => ({
+  root: {
```

Comment:
Noticed this is repeated in the style.css file. Why is that?

Reply:
Before using style.css, I was using 'makeStyles' Hook for all the styling, but the code started to look messy. So I copied all the styling from the component itself to the style.css. I still wanted to use some of the styling for <Paper>  using the hook. So I might have left some of the styling component in style.css which were not meant to be there.